uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -846,7 +846,7 @@ sub loadAndCreateObjectFile {
 
 Renames and moves the MINC file.
 
-INPUT:
+INPUTS:
```

Comment:
these inputs in the .md file looks weird (this function and 2 functions below);
please see screenshot (I think indentation is missing)

![indentationissues](https://user-images.githubusercontent.com/15198379/35992488-5ea5c9c2-0cd8-11e8-9be4-0b150cd95f1e.png)

Reply:
FYI: they look perfect if I run the perldoc command in  aterminal (just the .md one misses the new lines)